Doxyfile

```diff
@@ -829,7 +829,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = src/backend/BackendInterface.h
+INPUT                  = 
```

Comment:
hmm.. i think you want to leave the paths here. just don't generate html docs before you commit/push. 
i think in fact we need to specify something like src/* in here but please go read the doxy docs on how to specify a path for ALL your headers and cpp files. i'd prefer not to have to add a path to a new header every time we add a new one to the codebase.

Reply:
```
INPUT = src
RECURSIVE = YES
```